### ISSUE TITLE
navigation.openDrawer() sometimes not working

### ISSUE THREAD

**Issue by choo737** [react-navigation/react-navigation#5124]
There were two situations where navigation.openDrawer(); doesn't work.

1. When the app first loads, the user went into the login screen, after the user signed in with Facebook (With a dark overlay popped out) after navigated into the drawer screen.
2. When the app first loads, after the user automatically navigated into drawer screen when the token is persisted in Asyncstorage.

**Note: The drawer can be opened by swiping the top left corner.

![react navigation open drawer](https://user-images.githubusercontent.com/23527099/46913689-8ebb9d80-cfc4-11e8-85aa-f2b6021a081f.gif)

When I check on the redux states, the openDrawer only dispatched the OPEN_DRAWER and the next action (DRAWER_OPENED) is missing.

![image](https://user-images.githubusercontent.com/23527099/46913787-659c0c80-cfc6-11e8-9344-487778063db7.png)

My code is below:
```
import React from 'react';
import styled from 'styled-components/native';
import Ionicons from 'react-native-vector-icons/Ionicons';
import { connect } from 'react-redux';

const IconLeftContainer = styled.TouchableOpacity`
  height: 100%;
  paddingLeft: 15;
  justifyContent: center;
`;

const Hamburger = ({ navigation }) => (
  <IconLeftContainer onPress={() => {
    navigation.openDrawer();
  } }>
    <Ionicons name="ios-menu" size={25} color='white' />
  </IconLeftContainer>
);

export default connect()(Hamburger);
```
### Expected Behavior
The drawer should be able to open as usual when the app first loads, and after Facebook login.

### Your Environment
| software         | version
| ---------------- | -------
| react-navigation | 2.18.0
| react-native     | 0.55.4
| node             | 10.10.0
| yarn      | 1.9.4
| react     | 16.3.0

**Comment by choo737** [comment]
@eriveltonelias took me a while to reproduce the issue, please refer to this [snack](https://snack.expo.io/@choo737/ZHJhd2), only happen in Android.

**Comment by brentvatne** [comment]
I changed the implementation of this in the latest version of react-navigation-drawer, you can use it now if you want by installing the latest alpha version of it and importing from that package manually, or wait until it comes out of alpha when we release react-navigation v3 in a week or two.

**Comment by choo737** [comment]
@brentvatne Thanks! We've decided to skip the drawer entirely at the moment.. Will upgrade to react-navigation v3 when we need the drawer again. Thanks!

**Comment by jundat95** [comment]
*This can be a problem that you must be right?*

react-navigation@2.18.2
react-native@0.57.4

Please try

Options 1:

Open folder `node_modules/react-native-drawer-layout-polyfill/dist`
Delete file `index.android.js`

Option 2: 

copy folder `node_modules/react-native-drawer-layout-polyfill/` to `node_duplicate/react-native-drawer-layout-polyfill/`
install cpx: `npm i cpx --save-dev`
add scripts in package.json: `"copy": "cpx node_duplicate/react-native-drawer-layout-polyfill node_modules/react-native-drawer-layout-polyfill",`

Before build project run command auto replace: `npm run copy`

**Comment by brentvatne** [comment]
resolved in 3.0-rc.0

**Comment by sangnh1418** [comment]
hi! I did not find a solution from now.i use expo with react-navigation v5. Please help me thanks.
